README.md

```diff
@@ -89,6 +89,7 @@ version of TensorFlow I/O according to the table below:
 
 | TensorFlow I/O Version | TensorFlow Compatibility | Release Date |
 | --- | --- | --- |
+| 0.11.0 | 2.1.x | Jan 10, 2019 |
```

Comment:
Is 2.0.x still supported though? Backwards compatibility for major TF versions should be ensured IMO.

Reply:
@terrytangyuan The 2.0.x support may or may not work, due to the nature that we have to compile against C++ API in 2.1.x TensorFlow Core.

It might work (if there is no major C++ header file change) but that was never tested.

In the long run, once core tensorflow releases C API (ongoing) then we could guarantee backward compatibility.

Reply:
Yea it would be better to guarantee that in the future as it might not be as easy to upgrade TensorFlow for certain companies and then they won't be able to use these new features.